### PR TITLE
Gml dict

### PR DESCRIPTION
**Pull request by kwkwan**
### Metanorma PR checklist

 - [ ] Breaking changes (list related PRs)
 - [ ] Documentation update required ([create task for this](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] External dependency introduced ([documentation update need](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] Gem with native library introduced

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,4 +6,10 @@ git_source(:github) { |repo| "https://github.com/#{repo}" }
...
 gemspec
 
-eval_gemfile("Gemfile.devel") rescue nil
+begin
+  eval_gemfile("Gemfile.devel")
+rescue StandardError
+  nil
+end
+
+gem "ogc-gml", github: "lutaml/ogc-gml"
```
Comment (ronaldtse) [inline Gemfile:15]: @kwkwan please move this to the gemspec

File: README.adoc [hunk 2]
```diff
@@ -293,6 +293,42 @@ render it as:
 image::{{ image_base_path }}/{{ diagram.xmi_id }}.{{ format | default: 'png' }}[]
 -----
 
+=== Rendering a LutaML GML Dictionary: `lutaml_gml_dictionary`
+
+This command allows to render a LutaML GML Dictionary.
+
+[source,adoc]
+-----
+lutaml_gml_dictionary::[xml_path="/path/to/dictionary.xml",source="source_string",template_path="/path/to/template.liquid"]
```
Comment (ronaldtse) [inline README.adoc:302]: Please support these two syntaxes for the user document:

The first method allows the user to write the liquid template inline.
```adoc
[lutaml_gml_dictionary,"iur/codelists/3.1/BuildingInstallation_class.xml",root]
----
== GML Dictionary for {{ filename }}

{% render 'gml_dict_template', root: root, filename: filename %}
----
```

The second method allows the user 
```adoc
lutaml_gml_dictionary::["iur/codelists/3.1/BuildingInstallation_class.xml",context=root,template="/path/to/template.liquid"]
```

Is it difficult to support?

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (kwkwan) [reply Gemfile:15]: @ronaldtse The gem `ogc-gml` has not been released yet.

**Comment by kwkwan** [comment]
This macro is needed to be registered in `metanorma-standoc`. (https://github.com/metanorma/metanorma-standoc/pull/903)

File: README.adoc (hunk 2 above)
Comment (kwkwan) [reply README.adoc:302]: I will investigate this.

**Review by ronaldtse — APPROVED** [review]

**Pull request merged**